README.md

```diff
@@ -1,7 +1,7 @@
 # Acorn
 
 [![Build Status](https://travis-ci.org/ternjs/acorn.svg?branch=master)](https://travis-ci.org/ternjs/acorn)
-[![NPM version](https://img.shields.io/npm/v/acorn.svg)](https://www.npmjs.com/package/acorn)  
+[![NPM version](https://img.shields.io/npm/v/acorn.svg)](https://www.npmjs.com/package/acorn)
```

Comment:
This rather looks like line trimming. In Markdown, two extra spaces in the end of the line are used for actual newline, so line trimming should be disabled.